### ISSUE TITLE
Invalid Return Signature on Call Java Void function from Rust 

### ISSUE THREAD

**Issue by Lucas-Palomo** [giovanniberti/robusta#21]
Hello, First of all, thank you very much for this 'library'.

I am trying to call a void function from rust, but the signature from method is wrong

**This is the output** 

```
> Task :Main.main() FAILED
	Call from Rust getWorld method
Error while throwing Java exception: Java exception was thrown

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.6.1/userguide/command_line_interface.html#sec:command_line_warnings
4 actionable tasks: 1 executed, 3 up-to-date
Exception in thread "main" java.lang.NoSuchMethodError: Linternal/Hello;.print()Ljava/lang/Object;
	at internal.Hello.world(Native Method)
	at main.Main.main(Main.java:12)

FAILURE: Build failed with an exception.
	
```

I think this problem is in the "Ljava/lang/Object" return.   

Would it be related to this issue https://github.com/giovanniberti/robusta/issues/8 ? Maybe is time to release a new version. 

**main/Main.java**

```java
package main;

import internal.Hello;

public class Main {
    public static void main(String[] args) {

        System.load("/home/palomo/libs/libcore_serve.so");

        // Java
        Hello hello = new Hello();
        System.out.println(hello.world());

    }
}
```

**internal/Hello.java**

```java
package internal;

public class Hello {

    public native String world();

    public String getWorld() {
        System.out.println("\tCall from Rust getWorld method");
        return "World";
    }
    public void print() {
        System.out.println("\tCall from Rust print method");
    }

}
```

**lib.rs**

```rust
extern crate core;

use robusta_jni::bridge;

#[bridge]
mod jni {
    use robusta_jni::convert::{Signature, TryIntoJavaValue, TryFromJavaValue};
    use robusta_jni::jni::JNIEnv;
    use robusta_jni::jni::objects::{AutoLocal, JValue};
    use robusta_jni::jni::errors::Result as JniResult;
    use robusta_jni::jni::errors::Error as JniError;


    #[derive(Signature, TryIntoJavaValue, TryFromJavaValue)]
    #[package(internal)]
    pub struct Hello<'env: 'borrow, 'borrow> {
        #[instance]
        raw: AutoLocal<'env, 'borrow>,
    }

    impl<'env: 'borrow, 'borrow> Hello<'env, 'borrow> {
        pub extern "jni" fn world(self, env: &JNIEnv) -> String {
            self.getWorld(env);
            self.print(env);
            return "Hello World".to_string();
        }

        pub extern "java" fn getWorld(&self, env: &JNIEnv) -> JniResult<String> {}

        pub extern "java" fn print(&self, env: &JNIEnv) -> JniResult<()> {}
    }


}
```

**Comment by Lucas-Palomo** [comment]
I solved this problem by using the git version instead of the crate version

**Cargo.toml**
```diff
[dependencies]
- robusta_jni = "0.2"
+ robusta_jni = { git = "https://github.com/giovanniberti/robusta.git" }
```

I think it is time to release a new version of this crate

**Comment by giovanniberti** [comment]
Hi, thank you for the issue! I've just released version `0.2.1`, so you can use that instead.
